appcelerator/titanium_mobile
# fix(android): native modules should use min sdk for dependency

- Prevent native Titanium Android modules from depending on current `org.appcelerator.titanium` version and use `minsdk` specified in module manifest

##### TEST CASE
1. Build a native Android module (i.e: https://github.com/appcelerator-modules/titanium-identity)
2. Include module in project that uses an older Titanium SDK
3. Build should not fail with
```
[ERROR] : [GRADLE] Could not determine the dependencies of task ':app:compileDebugJavaWithJavac'.
[ERROR] : [GRADLE] > Could not resolve all task dependencies for configuration ':app:debugCompileClasspath'.
[ERROR] : [GRADLE] > Could not find org.appcelerator:titanium:9.0.3.
```

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-28092)
